src/cliques.jl

```diff
@@ -37,24 +40,32 @@ function pothen_sun{T<:Integer}(par::Vector{T}, post::Vector{T}, colcount::Vecto
     ns, flag
 end
 
-function chordal_embedding{Tv<:Number,Ti<:Int}(A::SparseMatrixCSC{Tv,Ti}, Perm::Vector{Ti})
+function round_array(T, A)
```

Comment:
why in the definition do you have only the round for Integer? The goal of `round` in Julia 0.6 is to make the type as T.

Reply:
Julia 1.5 fails to find a matching method when T is not a subtype of Integer.

Good point about forcing the type to be T though. I guess a manual explicit conversion is called for.

Reply:
New commit converts the type explicitly.